Added function for removing stopping words

redhed976 opened the pull request.

redhed976 committed:
Added function for removing stopping words

redhed976 committed:
Merge branch 'master' into create-stopping-word-removal

redhed976 committed:
Added removeStoppingWords to further clean body_text and abstract

darkhark commented on preprocessing/DataCleanser.py:
```diff
@@ -1,7 +1,12 @@
+import nltk
+from nltk.corpus import stopwords
+nltk.download('punkt')
```
This only needs to be downloaded once, so I don't think we need it on our code. Maybe just comment it out and say if needed, uncomment to download. Same applies to stopwords.
Also, what does punkt do? I get downloading the stop words, but not this.

darkhark commented on preprocessing/DataCleanser.py:
```diff
@@ -15,7 +20,7 @@ def handleEmptyData(df_covid):
     for col in df_covid.columns:
         if col == 'abstract':
-            df_covid['abstract'].replace('', 'no abstract provided', inplace=True)
+            df_covid['abstract'].replace('', 'Abstract Missing', inplace=True)
```
We kept this lowercase to avoid extra work later on when making everything lowercase.

darkhark commented on preprocessing/DataCleanser.py:
```diff
@@ -51,6 +56,21 @@ def removePunctuation(df_covid):
+    """
+    stop = stopwords.words('english')
+    pat = r'\b(?:{})\b'.format('|'.join(stop))
+    #print(pat)
```
Remove unneeded comments

darkhark commented on preprocessing/DataLoader.py:
```diff
@@ -131,4 +131,4 @@ def runDataLoader():
 def runQuickLoader():
     all_json, metadata = getAllData()
-    return getDataFrame(all_json[:1000], metadata)
+    return getDataFrame(all_json[:100], metadata)
```
This is fine, but if you make changes just for your own testing, try not to push those changes.

darkhark commented on Main.py:
```diff
@@ -2,8 +2,10 @@
-covidDF = runQuickLoader()
+covidDF = runDataLoader()
+#covidDF = runQuickLoader()
+print("\n---------Abstract after initial load-------------\n")
```
This happens twice

darkhark commented on preprocessing/DataCleanser.py:
```diff
@@ -1,7 +1,12 @@
+from nltk.corpus import stopwords
+nltk.download('punkt')
+nltk.download('stopwords')
+from nltk.tokenize import word_tokenize
```
Is this used anywhere?

redhed976 committed:
Removed duplicate print statements from Main.py

Removed unneeded imports
Commented out download statement for stopwords
Changed upper to lower case in abstract.replace
Removed unneeded commented out code
Reverted runQuickLoader parameter to 1000 records
